packages/mui-joy/src/Menu/Menu.tsx

```diff
@@ -148,6 +132,27 @@ const Menu = React.forwardRef(function Menu(inProps, ref) {
     [modifiersProp],
   );
 
+  const rootProps = useSlotProps({
+    elementType: MenuRoot,
+    getSlotProps: getListboxProps,
+    externalForwardedProps: other,
+    externalSlotProps: {},
+    ownerState: ownerState as MenuOwnerState & ListOwnerState,
+    additionalProps: {
+      ref,
+      anchorEl,
+      open,
+      disablePortal,
+      keepMounted,
+      as: PopperUnstyled,
+      modifiers,
+      slots: {
+        root: component || 'ul',
+      },
```

Comment:
Can you add a test to ensure that the `Menu` produces `ul`?

Reply:
Addressed in https://github.com/mui/material-ui/pull/36599/commits/36231c4464187b66adb8634947503f7aa8515046